alb-shared-to-k8s.tf

```
@@ -175,3 +175,14 @@ resource "aws_security_group_rule" "batcave_alb_shared_egress" {
   cidr_blocks       = ["0.0.0.0/0"]
   security_group_id = aws_security_group.batcave_alb_shared[0].id
 }
+
+data "aws_wafv2_web_acl" "cms_waf" {
+  name  = "RegSamQuickACLEnforcingV2"
+  scope = "REGIONAL"
+}
+
+resource "aws_wafv2_web_acl_association" "cms_waf_assoc" {
```

Name conflict for the same context reasons
```suggestion
resource "aws_wafv2_web_acl_association" "batcave_alb_shared_cms_waf_assoc" {
```

---

That was lifted verbatim from the alb-public-to-k8s.tf so for naming convention should that be changed too?

---

Right, you can't have 2 resources named the same thing.  It's just gotta be different.  Naming convention change would be great, but  that would mean resource migration which is annoying.